Sources/Build/BuildPlan.swift

```diff
@@ -140,6 +140,9 @@ public struct BuildParameters {
     /// `.swiftmodule`s.
     public let enableParseableModuleInterfaces: Bool
 
+    /// Maximum number of threads to use for compilation (0 means use number of CPUs)
+    public let numThreads: UInt32
```

Comment:
Does 0 have same semantics in the compiler? /cc @jrose-apple

Reply:
We don't document any such thing.

Reply:
@TaborKelly Let's not give 0 a special meaning in SwiftPM then.

Reply:
This is actually passed to llbuildSwift as `schedulerLanes` (or `'--jobs` on the **llbuild** command line) which is already relying on 0 meaning all. With that said an optional might be better, I'm happy to change it.